DistFiles/xMatter/bloom-xmatter-mixins.jade

```diff
@@ -64,11 +64,9 @@ mixin block-licenseAndCopyright
 		.copyright(data-book="copyright", lang="*").Credits-Page-style
 			{copyright}
 		.licenseBlock
-			img.licenseImage(src="" data-book="licenseImage")
-			.licenseUrl(data-book="licenseUrl", lang="en")
-				{License Url}
-			.licenseDescription(data-book="licenseDescription",lang="en").Credits-Page-style
-				{License Description}
+			img.licenseImage(src="license.png" data-book="licenseImage")
+			.licenseUrl(data-book="licenseUrl", lang="en") http://creativecommons.org/licenses/by/4.0/
+			.licenseDescription(data-book="licenseDescription",lang="en").Credits-Page-style You are free to make commercial use of this work. You may adapt and add to this work. You must keep the copyright and credits for authors, illustrators, etc.
```

Comment:
When the book is opened for editing, the string gets localized automatically (assuming a translation exists, of course) according to my tests.  Since the book is opened for editing automatically when created, the user would see the English text only if a translation doesn't exist.  I don't know of any way to access the .tmx files from jade.  If there is a way to do so, and a way to easily determine which language the book is in, then it might be worth doing.  Otherwise, the current behavior may be good enough.